src/core/oned/AbstractUPCEANReader.ts

```diff
@@ -195,7 +195,10 @@ export default abstract class AbstractUPCEANReader extends OneDReader {
   static findGuardPattern(row: BitArray, rowOffset: number, whiteFirst: boolean, pattern: Int32Array, counters: Int32Array): Int32Array;
   static findGuardPattern(row: BitArray, rowOffset: number, whiteFirst: boolean, pattern: Int32Array, counters?: Int32Array): Int32Array {
     if (typeof counters === 'undefined') counters = new Int32Array(pattern.length);
+    return this.findGuardPatternOverload(row, rowOffset, whiteFirst, pattern, counters);
+  }
 
+  static findGuardPatternOverload(row: BitArray, rowOffset: number, whiteFirst: boolean, pattern: Int32Array, counters: Int32Array): Int32Array {
```

Comment:
Now that I saw this it got me thinking: should we adopt something like `<methodName>Impl()` like some other languages do? That makes sense?

Reply:
Kinda what I had first with `findGuardPatternImplementation`, but I am happy with `findGuardPatternImpl`. I will merge this after the change.

Reply:
Yeah I chose the `<methodName>_Overload[1-9]` in a hurry, don't think that that was a good approach, I should've gone with the `Implementation` word from the start, but `Impl` seems good.